spec/models/spree/variant_spec.rb

```diff
@@ -987,11 +987,11 @@
 
       variant.variant_unit = 'weight'
       variant.variant_unit_scale = 1
-      variant.variant_unit_name = 'g'
```

Comment:
I think we need to remove this step as per the logic described in the issue that only variants with unit equal `items` have the value for `variant_unit_name`, otherwise it should be empty.